domaintools_async/__init__.py

```diff
@@ -4,18 +4,24 @@
 
 from domaintools.base_results import Results
 
+from domaintools.exceptions import ServiceUnavailableException
```

Comment:
I apologize, I lost this comment from my initial review: I may be missing something, but I don't currently see this import used. I just want to ensure this doesn't allude to any additional work you would like to include in this pull request